src/app/services/global-error-handler.service.ts

```diff
@@ -0,0 +1,16 @@
+import { ErrorHandler, Injectable } from '@angular/core';
+
+@Injectable()
+export class GlobalErrorHandler implements ErrorHandler {
+    handleError(error: Error) {
+        const chunkFailedMessage = /Loading chunk [\d]+ failed/;
+
+        if (chunkFailedMessage.test(error.message)) {
+
+            if (confirm('New version available. Load New Version?')) {
```

Comment:
Do we need to confirm? I feel like it might be better UX if we just automatically reload